dataland-internal-storage/src/test/kotlin/db/migration/V9__MigrateRefactoredDataPointClassesEuTaxNonFinancialsTest.kt

```diff
@@ -7,6 +7,7 @@ class V9__MigrateRefactoredDataPointClassesEuTaxNonFinancialsTest {
     @Test
     fun `test migration of refactored data point classes in eu taxonomy non financials framework`() {
         TestUtils().testMigrationOfSingleDataset(
+            "eutaxonomy-non-financials",
```

Comment:
Why are the older migration scripts modified?

Reply:
See answer above

Reply:
ok